src/client/containers/PullRequestSummary/Header/__tests__/index.js

```diff
@@ -0,0 +1,41 @@
+import chai from 'chai'
+import {
+  getHeaderData,
+} from '../index'
+import { types } from 'ducks/pullrequests/actions'
+
+const expect = chai.expect
+
+describe('getHeaderData', () => {
```

Comment:
Please, name test case properly and test also negative scenarios(no pull request, no users)